kpops/components/base_components/kafka_app.py

```diff
@@ -85,6 +85,12 @@ def clean_up_helm_chart(self) -> str:
         """Helm chart used to destroy and clean this component."""
         raise NotImplementedError
 
+    @property
+    def clean_up_release_name(self) -> str:
+        suffix = "-clean"
+        # TODO: Should this be self.helm_release_name + suffix ?
+        return create_helm_release_name(self.full_name + suffix)
```

Comment:
This used to be `self.helm_release_name + suffix`. But my assumption is that this would be fine like this. I just wanted to point it out.

Reply:
the `helm_release_name` property allows customization, therefore I would argue it would be more consistent to keep it here, too